scout/server/extensions/clinvar_extension.py

```diff
@@ -15,9 +15,9 @@ class ClinVarApi:
     the ClinVar submission schema (https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/) and if valid, can be directly submitted to ClinVar.
     """
 
-    def __init__(self):
+    def init_app(self, app):
         self.convert_service = "/".join([PRECLINVAR_URL, "csv_2_json"])
-        self.submit_service = CLINVAR_API_URL
+        self.submit_service = app.config.get("CLINVAR_API_URL") or CLINVAR_API_URL
```

Comment:
The name is good, but it is always a little confusing when the same name in different name spaces, that both are in use here. Consider something like `CLINVAR_API_URL_DEFAULT` for the const?

Reply:
Fixed! I'm going to open a request in servers so We have the apitest in scout stage